terms/demographics/ethnicity.json

```diff
@@ -0,0 +1,6 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "sage.annotations-demographics.ethnicity-0.0.1",
+  "type": "string",
```

Comment:
If we are going to have an enumerated list for race, should we also have one for ethnicity?

Reply:
The race categories are dictated by the NIH and have a discrete number of values.  Every ethnicity list I've found has had a large number of possible values, and are seldom consistent.  For example, some lists just list "Jewish" as an ethnicity, and some will actually break it down into separate subgroups like "Ashkenazi", "Sephardic", etc.  There are also a large number of African and Turkish/Middle Eastern ethnic identities represented in the US and elsewhere that we'd have to account for. The list would get pretty big pretty quickly and we'd probably still be missing some.

Reply:
With that said, the official governmental ethnicity list has two options:

1) Hispanic/Latino
2) Non-Hispanic/Non-Latino

If this is all we care about, we could make it an enumerated list.  This might be inadequate for future work with consortium studying diseases that are more prevalent in certain ethnic groups.

Reply:
I agree with @cw-dvr8 s concern about being prescriptive about ethnicity. The hispanic vs non-hispanic will probably not be sufficient for VEOIBD. Rush has a third category called Spanish ethnicity (Spanish/Hispanic/Latino origin) which is bolean. Can we adopt something similar and leave ethnicity as free text?

Reply:
I think that would be a good compromise.  If we ever need to do any reporting that involves the official ethnicity list, it would be pretty easy to generate Hispanic/non-Hispanic from a boolean.